torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2212,6 +2221,27 @@ def test_set_and_get_num_send_recv_threads(self):
         self.assertEqual(int(info["agent.thread_pool_size"]), NUM_THREADS)
         rpc.shutdown()
 
+    # FIXME Merge this test and the previous one
+    @dist_init(setup_rpc=False)
+    @_skip_unless_tensorpipe_agent
```

Comment:
@osalpekar would I be correct if I assume that moving these tests to `class TensorPipeAgentRpcTest` will save use this skip decorator?

Reply:
Yes, we should move the TP-only tests to `class TensorPipeAgentRpcTest`